SR2024/2023-09-27-place-rejection.md

```diff
@@ -0,0 +1,19 @@
+---
+to: Student Robotics 2024 rejections
+subject: Student Robotics 2024 is sadly full
+---
+
+Hello!
+
+We had a lot more interest in Student Robotics 2024 than we had places available. This has meant that unfortunately we're unable to offer you a place this year.
+
+We realise this is disappointing (we're disappointed too!), however we are limited by the number of kits we have and by the size of the venue. We are always looking for ways to support more teams and hope to be able to do so in future.
+
+If we have any early drop outs, we will be sure to let you know as you may be able to take their place, but we cannot promise anything.

```

Comment:
Explicitly mentioning the words "waiting list" would be good here.

Reply:
Happy with the below?
```suggestion
We've put you on our waiting list so if we have any early drop outs we will be sure to let you know. You may be able to take their place, but we cannot promise anything.
```